servers/xr/xr_interface.h

```diff
@@ -167,6 +171,9 @@ class XRInterface : public RefCounted {
 	virtual RID get_vrs_texture(); /* obtain VRS texture */
 	virtual VRSTextureFormat get_vrs_texture_format() { return XR_VRS_TEXTURE_FORMAT_UNIFIED; }
 
+	/** visionOS foveation support. No need to expose this method in GDScript **/
+	virtual void *get_rasterization_rate_map() { return nullptr; }
```

Comment:
Would it be possible to use `get_vrs_texture()` and `get_vrs_texture_format()` for this, even if it means adding a new `VRSTextureFormat` type? I think that would be better than adding a virtual method that's only for VisionOS

It's `get_vrs_texture()` that's used to provide density maps (or shading rate maps) for foveation on other platforms

Reply:
Thanks for the review, great idea, I have done this.

Reply:
Thanks! Looks good :-)